docs/reference/errors-and-warnings/NU1605.md

```diff
@@ -12,6 +12,8 @@ f1_keywords:
 
 # NuGet Warning NU1605
 
+[!INCLUDE [nugetsolver-tool](../../includes/nugetsolver-tool.md)]
```

Comment:
There're several different cases and their specific solutions are listed below, but adding this one under each one is bit repetitive. Also in the bottom there is `Note` section already, that's why I placed on top here, if you have better idea then please let me know.